roles/lb_haproxy/templates/haproxy.cfg.j2

```diff
@@ -84,7 +84,7 @@ frontend fe_https
 
     # insert strict CSP if none was set
     acl has_csp_header res.hdr(Content-Security-Policy) -m found
-    http-response set-header Content-Security-Policy "default-src 'self'; frame-src 'none'; form-action 'self' http://*.{{base_domain}}; frame-ancestors 'none'; block-all-mixed-content;" if !has_csp_header
+    http-response set-header Content-Security-Policy "default-src 'self' 'unsafe-inline'; frame-src 'none'; form-action 'self' http://*.{{base_domain}}; frame-ancestors 'none'; block-all-mixed-content;" if !has_csp_header
```

Comment:
Dat lijkt me wel een heel ruime default.  Ik zou liever zien dat we (bv voor sota en pyff) in de applicaties de ruime CSP zetten (als dat nodig is) ipv in de default voor alle hosts.
Voor welke applicatie is deze nodig?

Reply:
Inline is SATOSA, images is pyFF